tests/apps/core/test_templatetags_feature_flags.py

```diff
@@ -0,0 +1,52 @@
+"""
+Unit tests for the template tags related to FeatureFlags.
+"""
+from django.test import TestCase
+from django.test.utils import override_settings
+
+from richie.apps.core.templatetags.feature_flags import is_feature_enabled
+
+
+class FeatureFlagsTemplateTagsTestCase(TestCase):
+    """
+    Unit test suite to validate the behavior of the joanie template tags
+    """
+
+    @override_settings(
+        FEATURES={
+            "TEST_FEATURE": True,
+        }
+    )
+    def test_templatetags_features_enabled(self):
+        """
+        features should return True if
+        an feature flags is set to True in settings.
+        """
+        self.assertTrue(is_feature_enabled("TEST_FEATURE"))
```

Comment:
Do not forgot to rename test suite and docstrings according to the new template tag name.

```suggestion
    def test_templatetags_is_feature_enabled_true(self):
        """
        `is_feature_enabled` should return True if
        an feature flags is set to True in settings.
        """
        self.assertTrue(is_feature_enabled("TEST_FEATURE"))
```